TrainingRNExpress/src/app/components/Book/index.js

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { View, Text, Image } from 'react-native';
+import { View, Text, Image, TouchableOpacity } from 'react-native';
 
 import { RED, PLACEHOLDER_IMAGE } from '../../../constants';
 
 import styles from './styles.js';
 
-function Book({ image, title, author }) {
+function Book({ id, image, title, author, navigation }) {
   return (
-    <View style={styles.container}>
+    <TouchableOpacity style={styles.container} onPress={() => navigation.navigate('Book', { id })}>
```

Comment:
I'd put this literal at route constants

Reply:
I'd extract this function as an instance method so we avoid creating a new one on every render